skcosmo/selection/CUR.py

```diff
@@ -0,0 +1,297 @@
+from abc import abstractmethod
+import numpy as np
+from scipy.sparse.linalg import eigs as speig
+
+from skcosmo.pcovr.pcovr_distances import get_Ct, get_Kt
+from .orthogonalizers import feature_orthogonalizer, sample_orthogonalizer
+
+
+class _CUR:
```

Comment:
```suggestion
class BaseCUR:
```

or 

```suggestion
class CURBase:
```

Reply:
I had intended that _CUR and _FPS never be called directly, hence the `_`. Would you be good with `_BaseCUR`?

Reply:
Sure, that's better!